%YAML:1.0
---
StepMotor_dir: 17
StepMotor_interrupt: 13
StepMotor_Reset_Frequency: 800
motor_direction_flag: 1
Slide_block_compensation: 5.0